javascript-forms/form-events.js

```diff
@@ -0,0 +1,29 @@
+function handleFocus(event) {
+  console.log('focus event fired');
+  console.log(event.target.name);
+}
+
+function handleBlur(event) {
+  console.log('blur event fired');
+  console.log(event.target.name);
+}
+
+function handleInput(event) {
+  console.log('value of ' + event.target.name + ': ' + event.target.value);
+}
+
+var $userName = document.querySelector('label[for=user-name] + input');
+var $userEmail = document.querySelector('label[for=user-email] + input');
+var $userMessage = document.querySelector('label[for=user-message] + textarea');
```

Comment:
Since a `<label>`'s `for` attribute refers to an `id`, and the `id` is on the form control itself, it'd be easier to select the `id` directly (ex: `document.getElementById('user-name')`).